Sources/SymDiff/source/Config.cs

```diff
@@ -97,6 +100,12 @@ public ParamMap FindPair(string s1, string s2)
       return null;
     }
 
+    public bool PairEquivalent (string s1, string s2)
```

Comment:
This is never called from this codebase. Is the idea that it's just for external clients?

Reply:
Yes, this would be for external clients.  So I could add methods with names `SetEquivalent` and `GetEquivalent` to have the caching pop out.

Reply:
Given that this isn't used internally by this class at all, would it make more sense to have the client manage it entirely?

Reply:
No. This class serves as a channel between symdiff and eboo. Eboo stores in the cache and symdiff checks in the cache.

Reply:
Think of the Config as part of the input to symdiff. Config contains which methods are mapped to one another and also contains methods which are already assumed to be equivalent.

Reply:
I think the thing that made me ask the question is that, currently, SymDiff _doesn't_ check the cache. Is the plan that it will later?

Reply:
Oh I see. I forgot to commit the changes to symdiff skipping the procs in the PR.

Reply:
Added now, thanks

Reply:
Ah, good! That makes sense now. :)